### DISCUSSION TITLE
Remote Play Support (Airplay & Chromecast)

### DISCUSSION THREAD

**Discussion by kaambiz** [vidstack/player#891] [Q&A] [answered]
I looked everywhere yet i failed to find any resources on how to implement Chromecast and Airplay in Vidstack. Are they supported at this stage at all? I know they are available in icons. 

Regards.

**Comment by mihar-22** [comment, marked as answer]
Not yet @kaambiz, but we plan to add support for it shortly after 1.0.

**Reply by zrtsky** [reply]
Hello, When do you think you plan to release version 1.0, and would like to ask if there is a way to somehow implement airplay on my own?

I also wanted to say that I really like your idea about the player and the ways you have chosen to develop it 🙏

**Reply by mihar-22** [reply]
It's not too far away, I'm just building examples and then a little more docs before I do a 1.0 pre-release. Life is busy so I'm just a little slower than I like at the moment.

You can implement Airplay on your own like so:

```ts
// Check if airplay is supported:
if (typeof window.WebKitPlaybackTargetAvailabilityEvent === 'function') {
  // Call on video element to show Airplay dialog:
  video.webkitShowPlaybackTargetPicker();
}
```

You can get the video element using [events](https://www.vidstack.io/docs/player/providers/video#video-element).

**Reply by zrtsky** [reply]
Thanks a lot, this is help me a lot
